### DISCUSSION TITLE
Chat history can't be saved at this time.

### DISCUSSION THREAD

**Discussion by JohnixOS** [microsoft/sample-app-aoai-chatGPT#1054] [Q&A]
Hello, I migrated the Cosmos DB service to a new subscription, now I'm getting this message and I don´t know how to solve it:

Chat history can't be saved at this time. Error code: 404 - {'error': {'code': '404', 'message': 'Resource not found'}}

This occurs for any input I enter.

In my .env file I have all this variables:
AZURE_COSMOSDB_ACCOUNT=myAccount
AZURE_COSMOSDB_DATABASE=db_conversation_history
AZURE_COSMOSDB_CONVERSATIONS_CONTAINER=conversations
AZURE_COSMOSDB_ACCOUNT_KEY=myKey...==
AZURE_COSMOSDB_ENABLE_FEEDBACK=True

Any help is welcome.

Thanks.

**Comment by stbere** [comment]
Is the AZURE_OPENAI_RESOURCE variable in the same subscription as your Cosmos DB? That's the only thing I can think of
